stage1/gc/gc.go

```diff
@@ -47,10 +47,6 @@ func init() {
 func main() {
 	flag.Parse()
 
-	if !debug {
```

Comment:
I see that instead of using log package, we just do `fmt.Fprintf`ing to `os.Stderr`. Those `Fprintf`s probably could use a prefix, too.